docker/Dockerfile_ros-kinetic

```diff
@@ -18,13 +18,19 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6B
 		libeigen3-dev \
 		libgeographic-dev \
 		libopencv-dev \
+		libyaml-cpp-dev \
 		protobuf-compiler \
 		python-catkin-tools \
 		python-tk \
+		python3-pip \
+		python-wstool \
```

Comment:
I don't think we need wstool anymore.

Reply:
We pull the dependencies with wstool in avoidnace. Is this already included somewhere?